tachyon/zk/plonk/examples/BUILD.bazel

```diff
@@ -32,6 +32,19 @@ tachyon_cc_library(
 
 # TODO(dongchangYoo): This is failed in CI because of timeout, 60 secs.
 # Change |tachyon_cc_test| to |tachyon_cc_unittest| once CI can run within the timeout.
+tachyon_cc_test(
+    name = "shuffle_circuit_test",
+    srcs = ["shuffle_circuit_test.cc"],
+    deps = [
+        ":circuit_test",
+        ":shuffle_circuit",
+        "//tachyon/zk/plonk/halo2:pinned_verifying_key",
+        "//tachyon/zk/plonk/keys:proving_key",
+        "//tachyon/zk/plonk/layout/floor_planner:simple_floor_planner",
+        "//tachyon/zk/plonk/layout/floor_planner/v1:v1_floor_planner",
```

Comment:
Even though we have the same proof for now, I think there should be test for V1 other than `CreateProof` as well since `V1FloorPlanner`s logic might change in the future.

Reply:
The fact that tests need to be changed when `FloorPlanner` is modified remains the same as it is now.
Therefore, I believe it's acceptable to separate the tests when V1FloorPlanner is modified.